src/plugins/initiatives/initiativesDirectory.js

```diff
@@ -1,8 +1,17 @@
 // @flow
 
 import {type ReferenceDetector} from "../../core/references";
+import {type NodeAddressT, NodeAddress} from "../../core/graph";
 import {type Compatible, fromCompat, toCompat} from "../../util/compat";
 import {type InitiativeRepository, type URL} from "./initiative";
+import {initiativeNodeType} from "./declaration";
+
+// Note: fictional tracker address, we won't create a node for the files.
+const initiativeFilePrefix = NodeAddress.fromParts([
+  "sourcecred",
+  "initiatives",
+  "directory",
```

Comment:
the convention we're using in other plugins is that the third component is the node type (e.g. `ISSUE` or `PULL_REQUEST`). since the node types with this prefix are individual initiatives, maybe we should be consistent with `INITIATIVE`. it's certainly not a directory node type.

Reply:
Good idea, does make the address clearer. In this case it would be an `INITIATIVE_FILE`.

Reply:
Defined the subtype now.
https://github.com/sourcecred/sourcecred/blob/c48c924a9f5fcf667d8219af80713d5c54d112fc/src/plugins/initiatives/initiativesDirectory.js#L14